CHANGELOG.next.md

```diff
@@ -47,6 +47,7 @@ Thanks, you're awesome :-) -->
 * Extended `pe` fields added to experimental schema. #1256
 * Add `elf` fieldset to experimental schema. #1261
 * Add `threat.indicator` fields to experimental schema. #1268
+* Added `file.fork_name` field #1288
```

Comment:
Can this entry be moved above under `Unreleased > Schema Changes > Added` instead?